cypress/assets/unfetch.umd.js

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
Is this file in use? I think it is only used if the tests are running against fixtures. If not in use, then remove it (you can always re-add it later from one of the other repos. Or more likely, it will probably be part of cli-utils-cypress in the future)

Reply:
It's used in `server/visitWithUnfetchOverwriteFn`, which in turn overwrites the `visit` command. I don't know enough about fixtures to make this call. It's not using fixtures now, but will it ever?